manifest.json

```
@@ -1,18 +1,18 @@
 {
-  "vendor": "CHANGE_ME",
-  "name": "CHANGE_ME",
-  "version": "0.0.0",
-  "title": "CHANGE_ME",
-  "description": "CHANGE_ME",
+  "vendor": "vtexbr",
+  "name": "list-wallet",
+  "version": "0.0.1",
+  "title": "Wallet App",
```

Just a suggestion, but wouldn't it be better to rename it to `List Wallet`?
```suggestion
  "title": "List Wallet",
```
Just so it's clear that it's about the list app and not a generic wallet app, while also removing the redundant `app` keyword.

---

Great ideia
You're 100% right!!
Thanks :)